js/src/ui/SelectionList/selectionList.css

```diff
@@ -47,6 +47,14 @@
   filter: none;
 }
 
+.default {
+  background: rgba(255, 255, 255, 0.75);
+
+  &:hover {
+    background: initial;
```

Comment:
afaict this is not a valid value. `background` -> `background-color` and `initial` -> `transparent` would be spec-compliant.